hedera-node/src/main/java/com/hedera/services/txns/TransitionRunner.java

```diff
@@ -53,13 +54,14 @@
 			TokenGrantKycToAccount, TokenRevokeKycFromAccount,
 			TokenAssociateToAccount, TokenDissociateFromAccount,
 			TokenAccountWipe,
-			CryptoTransfer
+			CryptoTransfer,
+			TokenDelete
```

Comment:
Please revert all misc changes made on this file, except for `TokenDelete` addition